### ISSUE TITLE
Monkey wrench femoral neck

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6103]
**Preferred term label:**
Monkey wrench femoral neck

**Synonyms**
Monkey wrench appearance of femoral neck
Swedish key appearance of femoral neck
Monkey wrench (Swedish  key) configuration of the proximal femur 

**Definition (free text, please give PubMed ID)**
The femoral neck region shows medial metaphyseal beaking and a significant enlargement of the lesser trochanter (with some enlargement also of  the greater trochanter), producing a monkey wrench (Swedish  key)  configuration of the proximal femur. A monkey wrench refers to a type of adjustable wrench with one fixed and one adjustable jaw at right angles to a straight handle.

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0003366

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:618870    1/1  (PMID:27599773) and 1/1 (PMID:31325655)
OMIM:251450 (revise this entire entry)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added term
